app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
+    window.peopleList = new PeopleList
+    peopleList.fetch().then ->
+      $('.card').remove()
```

Comment:
Typically reaching directly to the global DOM with jquery is a bit of an antipattern and a leak of responsibility. 

typically, you would want to call into a view instance and reset some state, instead of manually changing the DOM state, since the router should not be concerned with this.

Here is a talk that I gave on managing state in views and exploring the concept of passing state down into views (a lot like what happens in react)
https://www.youtube.com/watch?v=CTr-tTwRH3o